docs/UserGuide.md

```diff
@@ -124,17 +125,36 @@ Format: `help`
 
 ### Finding a show: `find`
 
-Finds the specified show in Trackermon.
+**General Find**: Finds shows in Trackermon by matching the user's input across the name, status and tag parameters.
 
 Format: `find <KEYWORD>`
-* Finds the show with the specified `<KEYWORD>`.
+* Find shows with the specified `<KEYWORD>`.
 * The keyword refers to the input entered by the user after `find`.
-* The keyword **can be a word or number** such as hero, S1,..
-* If the keyword is a word, it must be complete. 
+* The keyword **can be a word or number** such as hero, S1,...
+* There must be at **least one keyword** and it **must not be empty**.
 
 Examples:
-* `find` followed by `attack` displays all the shows in the list that contain the keyword `attack`.
-* `find attack` followed by `delete 1` removes 1st show in results of `find` command.
+* `find shingeki` displays all the shows in the list that contain the keyword `shingeki` whether it is a name, status or tag.[put image here after ui update]
+* * `find shingeki no kyojin` displays all the shows in the list that contain the keyword `shingeki` **or** `no` **or** `kyojin` whether it is a name, status or tag.[put image here after ui update]
+
+**Precise Find** Finds shows in Trackermon by matching the user's input across name, status and tag parameters with prefixes.
+
+Format: `find n/[NAME] s/[STATUS] t/[TAG]`
+* **Within a single prefix**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all keywords that are input by the user in the specific prefix.
+* **Across multiple prefixes**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all the keywords across all prefixes.
+* There must be at **least one prefix field** and it **must not be empty**.
+
+Examples:
+* `find n/shingeki s/watching t/seinen` displays all the shows in the Trackermon's show list that contain **shingeki** in NAME, **watching** in STATUS, and **seinen** in TAG. [put image here after ui update]
+* `find n/Shutter Island s/watching` displays all the shows in the Trackermon's show list that contain **Shutter AND Island** in NAME, and **watching** in STATUS. [put image here after ui update]
```

Comment:
I feel that as a user, we do not need to know too much of the specification of how it searched.
`... contains **Shutter AND Island**` could be simplified to `... contains **"Shutter Island"**`

Reply:
Noted.